.github/workflows/gradle.yml

```diff
@@ -10,27 +10,46 @@ on:
     - cron: '0 3 * * *'
 
 jobs:
+  Sonar-Check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+
+      - name: Sonar Scan Project
+        uses: gradle/gradle-build-action@67421db6bd0bf253fb4bd25b31ebb98943c375e1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONARCLOUD_TOKEN }}
+        with:
+          arguments: sonar
+
   build:
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+
       - name: Set up JDK 11
```

Comment:
omit the "11"

Reply:
Is setup JDK 8 okay?

Reply:
At least it is correct and not wrong, like 11 was. :-) I am OK to leave it like this.

Maybe just saying "Set up JDK" would be enough, if one needs to know the version it is best to check the yml file for the truth. And then next time we change the JDK version we avoid the risk of having an outdated name again.

Reply:
That makes sense. I think when updating the Java version, it would be best to also update the stage name.